docs/sources/configure-client/language-sdks/python.md

```diff
@@ -42,7 +42,6 @@ pyroscope.configure(
   sample_rate         = 100, # default is 100
```

Comment:
@Rperry2174 should we remove auth_token from this example to not confuse users?
auth_token is only needed for og pyroscope, seems like we should not advertise it.